cmd/kubeadm/app/cmd/config_test.go

```diff
@@ -232,9 +232,9 @@ func TestImagesPull(t *testing.T) {
 	}
 
 	images := []string{"a", "b", "c", "d", "a"}
-	ip := NewImagesPull(containerRuntime, images)
+	imagesPull := NewImagesPull(containerRuntime, images)
 
-	err = ip.PullAll()
+	err = imagesPull.PullAll()
```

Comment:
no need to change this variable name?

basically PRs should only focus on the changes that are explained in the title and description.
cleanups and refactors should either be mentioned or best omitted to not add noise in the DIFF. :)

Reply:
ok agreed. how is the workflow here, do we create issues on the fly or at the end? 

imho the `ip` name is ambigous, for `imagesPull`. At beginning i was thinking about netWorking stuff :smile: